pkg/kubelet/container/runtime.go

```diff
@@ -88,9 +89,19 @@ type Runtime interface {
 	// KillPod kills all the containers of a pod. Pod may be nil, running pod must not be.
 	KillPod(pod *api.Pod, runningPod Pod) error
 	// GetPodStatus retrieves the status of the pod, including the information of
-	// all containers in the pod. Clients of this interface assume the containers
-	// statuses in a pod always have a deterministic ordering (eg: sorted by name).
+	// all containers in the pod. Clients of this interface assume the
+	// containers' statuses in a pod always have a deterministic ordering
+	// (e.g., sorted by name).
 	GetPodStatus(*api.Pod) (*api.PodStatus, error)
+	// GetRawPodStatus retrieves the status of the pod, including the
+	// information of all containers in the pod that are visble in Runtime.
+	GetRawPodStatus(uid types.UID, name, namespace string) (*RawPodStatus, error)
```

Comment:
Please comment this method is going to be rename to GetPodStatus along with deprecating previous GetPodStatus defined above.